core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -382,7 +382,8 @@ private[spark] object JsonProtocol {
 
   def executorInfoToJson(executorInfo: ExecutorInfo): JValue = {
     ("Host" -> executorInfo.executorHost) ~
-    ("Total Cores" -> executorInfo.totalCores)
+    ("Total Cores" -> executorInfo.totalCores) ~
+    ("Log Urls" -> mapToJson(executorInfo.logUrlMap))
```

Comment:
Actually, I was mistaken: we don't need backwards-compatibility tests because this ExecutorInfo class is new in 1.3.  We should revert my commit to add the compatibility tests, since it's just adds clutter now.